### DISCUSSION TITLE
Recreate a deleted organization but got error

### DISCUSSION THREAD

**Discussion by xhavit** [npm/feedback#1053] [Website]
I deleted an organization, but now I want to recreate it, but the name doesn't work, and the name isn't taken, what should I do now?

**Comment by xhavit** [comment]
Following the prompts on this page (https://www.npmjs.com/support), I submitted my application, but no one responded to me, Help me please.

**Comment by fecony** [comment]
Same here, deleted org and can't recreate it as it's not available anymore :/
Not intuitive behaviour from ux side, docs don't have any note on "delete organization" page
